### PR TITLE
PA plot: Speedups for zero wave

### PR DESCRIPTION
**Pull request by t-b**

### CODE REVIEW

File: Packages/MIES/MIES_PulseAveraging.ipf [hunk 1]
```diff
@@ -1628,7 +1628,7 @@ static Function PA_ZeroPulses(WAVE/WAVE set, STRUCT PulseAverageSettings &pa)
 	endif
 
 	Make/FREE/N=(DimSize(set, ROWS)) junkWave
-	MultiThread junkWave = ZeroWave(set[p]) + PA_UpdateMinAndMax(set[p])
+	MultiThread junkWave = ZeroWave(set[p]) && PA_UpdateMinAndMax(set[p])
```
Comment (timjarsky) [inline Packages/MIES/MIES_PulseAveraging.ipf:1631]: @t-b can we talk about this line the next time we chat?

Comment (timjarsky) [reply Packages/MIES/MIES_PulseAveraging.ipf:1631]: @timjarsky If zeroWave(set[p]) returns 1, then PA_UpdateMinAndMax(set[p]) is run

**Review by timjarsky — APPROVED** [review]

**Comment by MichaelHuth** [comment]
@t-b Please don't merge, this is not threadsafe and causes the Igor crashes I see. This is only safe without Multithread.

**Comment by timjarsky** [comment]
> @t-b Please don't merge, this is not threadsafe and causes the Igor crashes I see. This is only safe without Multithread.

@t-b uh oh, someone at Wavemetrics isn't going to be happy about this discovery ;)

**Comment by t-b** [comment]
I'm merging this now because:
- I does not provoke a crash as long as `set` does not have duplicates waves
- This code will be rewritten anyway in one of the new speedup PRs

**Pull request merged**